src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -149,6 +149,23 @@ export class CellModel extends Disposable implements ICellModel {
 		return this._attachments;
 	}
 
+	addAttachment(mimeType: string, base64Encoding: string, name: string): nb.ICellAttachments {
+		// base64Encoded value looks like: data:application/octet-stream;base64,<base64Value>
+		// get the <base64Value> from the string
+		let index = base64Encoding.indexOf('base64,');
+		if (index > -1) {
```

Comment:
Using a regex here seems better so you can verify the whole string is valid (right now if I pass in a string `test base64' then it'll try to parse it as a valid base64 string)